sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/api/Azure.ResourceManager.ConnectedVMwarevSphere.netstandard2.0.cs

```diff
@@ -0,0 +1,1760 @@
+namespace Azure.ResourceManager.ConnectedVMwarevSphere
```

Comment:
You can remove this file - I'm not sure why it is being generated.

/cc @m-nash @archerzz